resources/data.py

```diff
@@ -534,6 +537,8 @@ def fauna(chance: int = None, distance_below_sea_level: int = None, climate: Dic
 
 
 def food_item(rm: ResourceManager, name_parts: utils.ResourceIdentifier, ingredient: utils.Json, category: Category, hunger: int, saturation: float, water: int, decay: float, fruit: Optional[float] = None, veg: Optional[float] = None, protein: Optional[float] = None, grain: Optional[float] = None, dairy: Optional[float] = None):
+    if type(ingredient) != utils.Json:
+        rm.item_tag('tfc:pig_food', ingredient)
```

Comment:
This does not work how you think it does, and moreover, is just bad practice. Create a `tfc:foods` tag and let pig food reference that. Add all items explicitly to the food tag (a loop somewhere in the tags section).

you cannot `==` with types from the `typing` module. All the below return `True`:
```python
type('strings') != utils.Json
type({'dictionaries': 'with values'}) != utils.Json
type(None) != utils.Json
```
If you absolutely need to do runtime type checks, prefer `isinstance(obj, type)`, i.e. `isintance('thing', str)`. But you'll also find out that generic types cannot be type checked in this manner (since `utils.Json` means, "Any string, dictionary with string keys, or sequence of anything).